pkg/solidity-utils/test/WordCodec.test.ts

```diff
@@ -34,6 +34,9 @@ describe('WordCodec', () => {
       const MAX_VALID = getMaxValue(bits);
 
       expect(await lib.insertInt(ZERO_BYTES32, MAX_VALID, 0, bits)).to.equal(TypesConverter.toBytes32(MAX_VALID));
+      if (bits < 255) {
+        await expect(lib.insertInt(ZERO_BYTES32, getOverMax(bits), 0, bits)).to.be.revertedWith('CODEC_OVERFLOW');
+      }
```

Comment:
We should also validate the negative limits (which are not the same as the positive ones)

Reply:
This might be a stale commit you're looking at? It does validate -1 at every bit length.